src/lsm/schema.zig

```diff
@@ -4,40 +4,18 @@
 //! relevant parameters directly into the block's header. This allows the decoders to not be
 //! generic. This is convenient for compaction, but critical for the scrubber and repair queue.
 //!
-//! Every block begins with a `vsr.Header` that includes:
```

Comment:
Removing the header descriptions -- now that the `Metadata` struct is simple + uniform, I'd prefer that it be the "source of truth" for block header schemas.